catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/SeedCommandTest.java

```diff
@@ -38,12 +38,15 @@
 import java.util.function.Consumer;
 import java.util.function.Predicate;
 
+import org.apache.commons.lang3.math.NumberUtils;
```

Comment:
❗️ Delete as this is unused: causing Checkstyle error when running full build. Probably a residual from when you were using a ceiling function.